onyo/cli/cat.py

```diff
@@ -20,8 +20,8 @@
 
 
 def cat(args: argparse.Namespace) -> None:
-    """
-    Print the contents of **ASSET**\s to the terminal.
+    r"""
+    Print the contents of **ASSET**\ s to the terminal.
```

Comment:
I know you updated this based on what was there already -- however: I notice quite often the inconsistency between having the docstring begin in the line with the `"""` or in the line after that.

I think Ben pointed out a while ago, that these differences have actually a different meaning, where the first line is used by some documentation tools as a short description; whereas when begin in the second line it is considered part of the "long and complete" description.

That difference should be verified and unified along the code base at some point, though it might be out of scope for this PR and be done whenever as long as the API docs you are building right now can handle it properly :)